.travis.yml

```diff
@@ -30,7 +30,7 @@ before_install:
 install:
 
   - if [[ $MINIMAL_ENV == 'False' ]] ; then
-      DEPS="numpy scipy matplotlib ipython h5py sympy scikit-learn dill natsort setuptools scikit-image cython lxml ipyparallel";
+      DEPS="pytest pytest-cov numpy scipy matplotlib ipython h5py sympy scikit-learn dill natsort setuptools scikit-image cython lxml ipyparallel dask toolz";
```

Comment:
??? now it fails minimal Travis test. Why you remove it? Dask is stupid it will complain, it does not install it automatically when pip installing, and some parts of dask used needs this toolz, you dont get this on Anaconda environment as it is automagicaly silently is or even pre-installed (anaconda people knows that it is required )). Now if somebody will try to install this through the pip directly will run into same error Travis_minimal environment fails.